src/Components/Album.jsx

```diff
@@ -0,0 +1,13 @@
+import { AlbumName } from "./AlbumName.jsx";
+import { ArtistName } from "./ArtistName.jsx";
+import { CoverImage } from "./CoverImage.jsx";
+
+export const Album = (props) => {
```

Comment:
I suppose Antonella showed both approaches last week, but I just wanted to remind you that you can restructure the props for better readability. Have a look [here](https://www.notion.so/technigo/Week-5-React-Basics-Components-Props-af8d65bc028346caa8e20af9b3f22c94?pvs=4#b6d4c8c77ed444358cbd704d05af888d)